app/modules/individuals/models.py

```diff
@@ -610,7 +621,7 @@ def merge_notify(cls, individuals, request_data, notif_type=None):
             Individual._merge_notify_user(
                 current_user,
                 owner,
-                owners[owner]['individuals'],
+                individuals,
```

Comment:
I think the outcome from the discussoin with Mark was that this should not be done. Instead what is in my PR should be. 
If this is undone then I think the changes should avoid clashing